README.md

```diff
@@ -7,188 +7,80 @@ apuX platform users and developers to customize firmware to their needs.
-wget https://raw.githubusercontent.com/pcengines/apu2-documentation/master/ipxe/menu.ipxe -O $IPXE_PATH/src/menu.ipxe
-cd $IPXE_PATH/src
-make -j$(nproc) bin/8086157b.rom EMBED=./menu.ipxe
-```
-
-Feel free to customize `menu.pxe` and `local/general.h` to match your needs.
-
-Building sortbootorder
-----------------------
-
-> coreboot is in `./coreboot-${BR_NAME}` directory
->
-```sh
-git clone https://github.com/pcengines/sortbootorder.git sortbootorder
-cd sortbootorder
-# for mainline coreboot (4.5.x)
-COREBOOT_ROOT=../coreboot-${BR_NAME} make distclean
-COREBOOT_ROOT=../coreboot-${BR_NAME} make
-# for legacy coreboot (4.0.x)
-COREBOOT_ROOT=../coreboot-${BR_NAME} make distclean
-COREBOOT_ROOT=../coreboot-${BR_NAME} COREBOOT_REL=legacy make
-```
+Please note that you may have NFS server running on host what leads to ports
+conflicts.
 
-Building memtest86+
--------------------
+After starting NFS and HTTP you can boot apuX. Please enable network booting
+using [sortbootorder](https://github.com/pcengines/sortbootorder#theory-of-operation).
 
-```sh
-cd && cd coreboot-${BR_NAME}
-git clone https://github.com/pcengines/memtest86plus.git payloads/external/memtest86plus
-cd payloads/external/memtest86plus
-make -j$(nproc)
 ```
-
-cbfstool and adding/removing ROMs or payloads
----------------------------------------------
-
-`cbfstool` is result of `coreboot` build it gives ability to manipulate CBFS
-which filesystem for coreboot ROM.
-
-Usage examples:
-
-### Add iPXE ROM
-
-```sh
-cd && cd coreboot-${BR_NAME}
-./build/cbfstool ./build/coreboot.rom add -f payloads/external/ipxe/src/bin/8086157b.rom -n genroms/pxe.rom -t raw
+iPXE> ifconf net0
+iPXE> dhcp net0
+iPXE> chain http://<your_ip>:8000/menu.ipxe
 ```
 
-Above command add raw image (`8086157b.rom`, result of iPXE build) to
-`coreboot.rom` under the name `genroms/pxe.rom`. This makes SeaBIOS to auto
-detect iPXE ROM and execute it before entering menu.
-
-### Add sortbootorder
+Choose `Debian stable netboot 4.14.y` after boot login (`[root:debian]`) and
+run:
 
 ```
-# add sortbootorder
-cd && cd coreboot-${BR_NAME}
-./build/cbfstool ./build/coreboot.rom remove -n img/setup
-./build/cbfstool ./build/coreboot.rom add-payload -f payloads/pcengines/sortbootorder/sortbootorder.elf -n img/setup -t payload
+flashrom -p internal -w apuX_x.y.z.rom
 ```
 
-Above commands first remove already existing `img/setup` from CBFS and then add
-`sortbootorder.elf` as payload under the name `img/setup` to `coreboot.rom`.
-
+## Known issues
 
-### Add memtest86+
+### Incorrect privileges
 
 ```
-# add memtest86+
-cd && cd coreboot-${BR_NAME}
-./build/cbfstool ./build/coreboot.rom remove -n img/memtest
-./build/cbfstool ./build/coreboot.rom add-payload -f payloads/external/memtest86plus/memtest -n img/memtest - payload
+$ ./start.sh
+Error checking context: 'can't stat '/path/pxe-server/debian/debian-stable/var/lib/sudo''.         '
```

Comment:
We can also move the Dockerfile and scripts to a separate folder. This will prevent docker to yield such errors and fasten running the container. I will make a pull request for that change

Reply:
@miczyg1 I already fixed that problem, the solution is easier namely `.dockerignore` file